packages/gitgraph-core/src/commit.ts

```diff
@@ -29,7 +29,7 @@ export interface CommitOptions<TNode> extends CommitRenderOptions<TNode> {
  *
  * @return {string} hex string with 40 chars
  */
-const getRandomHash = () =>
+const getRandomHash = (): string =>
```

Comment:
🤔 

![image](https://user-images.githubusercontent.com/1761469/49500687-8f104080-f871-11e8-8875-0f9f2708797a.png)

Typescript already infer this kind of return, so I'm curious about the reason behind this 😄

Reply:
Just to make it explicit. But yeah, there was no problem with TS.